indico/modules/events/registration/fields/choices.py

```diff
@@ -108,7 +108,7 @@ def _check_number_of_places(form, field):
                     places_limit = choice.get('places_limit')
                     places_used_dict = self.get_places_used()
                     places_used_dict.setdefault(k, 0)
-                    if form.modified_registration:
+                    if form.modified_registration and old_data.data:
                         places_used_dict[k] -= (form.modified_registration
                                                     .data_by_field[self.form_item.id].data.get(k, 0))
```

Comment:
Question: Is written in this form on purpose to refetch the data jus there, or can this be written as  old_data.data.get...

Reply:
I was asking myself the same question when looking at this code ;)